package.json

```diff
@@ -64,11 +64,6 @@
     "npm-run-all": "^4.1.5",
     "rimraf": "^4.4.0"
   },
-  "overrides": {
```

Comment:
we may have CVEs against us if we remove this
my point was that this is the wrong package.json to modify, to correct one for the front-end is `td.vue/package.json`

Reply:
i got the point , will update the PR @jgadsden